lua/tests/gmod/globals/angle.lua

```diff
@@ -0,0 +1,168 @@
+return{
+    groupName = "Angle",
+
+    cases = {
+        --[[ Function Signature ]]--
+        --#region
+        {
+            name = "Function exists globally",
+            func = function()
+                expect( Angle ).to.beA( "function" )
+            end
+        },{
+            name = "Returns an Angle object",
+            func = function()
+                local angle = Angle( 1, 2, 3 )
+                expect( angle ).to.beA( "Angle" )
+            end
+        },
+        --#endregion
+        
+        --[[ Pitch/Yaw/Roll Variable Equivalents ]]--
+        --#region
+        {
+            name = "Pitch equivalents match input",
+            func = function()
+                local angle = Angle( 1, 2, 3 )
+                expect( angle.x ).to.equal( 1 )
+                expect( angle.p ).to.equal( 1 )
+                expect( angle.pitch ).to.equal( 1 )
+            end
+        },{
+            name = "Yaw equivalents match input",
+            func = function()
+                local angle = Angle( 1, 2, 3 )
+                expect( angle.y ).to.equal( 2 )
+                expect( angle.yaw ).to.equal( 2 )
+            end
+        },{
+            name = "Roll equivalents match input",
+            func = function()
+                local angle = Angle( 1, 2, 3 )
+                expect( angle.z ).to.equal( 3 )
+                expect( angle.r ).to.equal( 3 )
+                expect( angle.roll ).to.equal( 3 )
+            end
+        },
```

Comment:
Good tests.I also suggest using the numeric indices

```suggestion
        {
            name = "Pitch equivalents match input",
            func = function()
                local angle = Angle( 1, 2, 3 )
                expect( angle[1] ).to.equal( 1 )
                expect( angle.x ).to.equal( 1 )
                expect( angle.p ).to.equal( 1 )
                expect( angle.pitch ).to.equal( 1 )
            end
        },{
            name = "Yaw equivalents match input",
            func = function()
                local angle = Angle( 1, 2, 3 )
                expect( angle[2] ).to.equal( 2 )
                expect( angle.y ).to.equal( 2 )
                expect( angle.yaw ).to.equal( 2 )
            end
        },{
            name = "Roll equivalents match input",
            func = function()
                local angle = Angle( 1, 2, 3 )
                expect( angle[3] ).to.equal( 3 )
                expect( angle.z ).to.equal( 3 )
                expect( angle.r ).to.equal( 3 )
                expect( angle.roll ).to.equal( 3 )
            end
        },
```